webpack.config.js

```diff
@@ -147,6 +147,10 @@ const config = {
 			/debug/,
 			path.resolve(__dirname, "scripts/noop.js")
 		),
+		new webpack.ContextReplacementPlugin(
+			/vue-server-renderer/,
+			path.resolve(__dirname, "./src/vue-server-renderer")
```

Comment:
This doesn't resolve to anything useful. Just changing it to a random string also gets rid of the warning for some reason.

Any idea why that happens?

Reply:
I'm guessing it's just not specific enough to do what I think it's doing, so it's fixing the file path error by providing a static context for dynamic imports, but not in a way that it's limited to the specific module that's causing the error. I haven't worked with this type of webpack plugin before so it probably needs more finesse. 

I'm getting it to throw some new errors locally though, so I'll see if I can get it to work as intended.

Reply:
Did you get a chance to look at this?